lib/external/sentry.js

```diff
@@ -122,6 +124,10 @@ const init = (config) => {
     beforeSend(event, hint) {
       const sanitizedEvent = sanitizeEventRequest(event);
 
+      if (sanitizedEvent.extra && sanitizedEvent.extra.startTimestamp) { // just an extra safety check. this should be always true
```

Comment:
I think this won't be true for an error thrown by a worker. So a good check I think!